### DISCUSSION TITLE
Blog post about Next.js Incremental Static Regeneration

### DISCUSSION THREAD

**Discussion by arunoda** [vercel/next.js#14028] [Show and tell]
I created a blog post on [Next.js Incremental Static Regeneration](https://arunoda.me/blog/what-is-nextjs-issg). I tried to compare pure SSR and SSG approaches and how these improvements takes things further.

I also included a few demos inside the blog post to make things understandable.
I you still didn't understand how Next.js Incremental Static Regeneration works, I assume [this article](https://arunoda.me/blog/what-is-nextjs-issg) helps.

Let me know, what I need to improve or what I should change etc.

Have a nice day!

**Comment by danilockthar** [comment]
Hi arunoda. Great blog post. I followed the example and in my previous build with getstaticpath and getstaticprops i changed fallback : false, to true and added unstable_revalidate: 10 but this give me some error in the logs.

TypeError: Cannot read property 'title' of undefined
01:00:10.893  
    at Product (/vercel/74e16995/.next/serverless/pages/store/[slug].js:886:31)

If i go back to the previous config it works. What could be the problem ? i mean, in my function page i put the {data} of getstaticprops as prop.


```
export async function getStaticPaths() {
      const {data} = await allOverlays()
      return {
          paths: data.map(({slug}) => ({
            params: {slug}
          })),
          fallback: true
      }
  }


export async function getStaticProps({ params }) {
 const {data} = await bringOneOverlayBySlug(params.slug)
 return{
     props: {data},
     unstable_revalidate: 10
 }
```

**Reply by runoncedev** [reply]
Are you checking for `isFallback` in your component before trying to use the props?
